src/main/resources/mybatis-config.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+
+<!DOCTYPE configuration
+        PUBLIC "-//mybatis.org//DTD Config 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-config.dtd">
+<configuration>
+    <properties resource="db.pass"/>
```

Comment:
Is this another properties file? What does it contain?

Reply:
It is. It is not available on the GitHub repository  and the file were the information for the connection is pulled. the db.properties is just a decoy file